core/dbt/config/profile.py

```diff
@@ -401,9 +401,11 @@ def from_raw_profiles(
     @classmethod
     def render_from_args(
```

Comment:
should probably consider renaming this method now that it doesn't even take `args`. Maybe just `render`?

Reply:
Updated